dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -357,10 +378,13 @@ public void Test_Create_App_descriptor_Then_Create_App_Integration_Then_Delete_O
     @Test
     public void Test_Create_App_Descriptor_Then_Create_App_Integration_Then_Delete_App_Descriptor() {
 
-        final Map<String, Param> paramMap = ImmutableMap.of(
-                "param1", Param.newParam("val-1", false, Type.STRING, "label", "hint", true),
-                "param2", Param.newParam("val-2", false, Type.STRING, "label", "hint", true),
-                "param3", Param.newParam("val-3", false, Type.STRING, "label", "hint", true)
+        final SortedMap<String, ParamDescriptor> paramMap = ImmutableSortedMap.of(
+                "param1", ParamDescriptor
+                        .newParam("val-1", false, Type.STRING, "label", "hint", true),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "val-1" appears 12 times in this file; the first occurrence is on line 379](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)